e2e/admin-web/rbac/test_rbac_demo_user.py

```
@@ -0,0 +1,305 @@
+            print("\nScreenshots saved to /tmp/test_rbac_demo_*.png")
+
+            return True
+
+        except AssertionError as e:
+            print(f"\n[ASSERTION ERROR] {e}")
+            take_screenshot(page, "rbac_demo_error_assertion", "Assertion error")
+            import traceback
+
+            traceback.print_exc()
+            return False
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)